src/reporter/labels.json

```
@@ -73,5 +78,10 @@
     "name": "location",
     "color": "FBCA04",
     "description": "Fetch location is outdated"
+  },
+  {
+    "name": "invalid selector",
+    "color": "FBCA04",
+    "description": "Selector is invalid"
```

It should be plural, as there are several selectors in a declaration.

> I propose a rephrasing that uses different words than “invalid” to increase the amount of information provided to the user in case “invalid” is unclear.

```suggestion
    "description": "Some selectors cannot be understood by the engine"
```